src/lights/DirectionalLightShadow.js

```diff
@@ -1,25 +1,22 @@
 import { LightShadow } from './LightShadow.js';
 import { OrthographicCamera } from '../cameras/OrthographicCamera.js';
 
-function DirectionalLightShadow() {
+class DirectionalLightShadow extends LightShadow {
 
-	LightShadow.call( this, new OrthographicCamera( - 5, 5, 5, - 5, 0.5, 500 ) );
+	constructor() {
 
-}
-
-DirectionalLightShadow.prototype = Object.assign( Object.create( LightShadow.prototype ), {
-
-	constructor: DirectionalLightShadow,
+		super( new OrthographicCamera( - 5, 5, 5, - 5, 0.5, 500 ) );
 
-	isDirectionalLightShadow: true,
+	}
 
-	updateMatrices: function ( light ) {
+	updateMatrices( light ) {
 
-		LightShadow.prototype.updateMatrices.call( this, light );
+		super.updateMatrices( light );
 
 	}
 
-} );
+}
 
+DirectionalLightShadow.prototype.isDirectionalLightShadow = true;
```

Comment:
could you use the `Object.defineProperty()` pattern demonstrated in #20110 across the files modified in this PR? 🙏

Also support for class fields is coming soon.